.buildlibrary

```diff
@@ -1,8 +1,9 @@
-ValidationKey: '28082950'
+ValidationKey: '28109640'
 AcceptedWarnings:
 - 'Warning: package ''.*'' was built under R version'
 - 'Warning: namespace ''.*'' is not available and has been replaced'
 - '''.*'' is needed for checks on size reduction of PDFs'
+- was deprecated in ggplot2
```

Comment:
The new tests complain:
```
`aes_()` was deprecated in ggplot2 3.0.0.
ℹ Please use tidy evaluation idioms with `aes()`
───────────────────────────────────────────
`aes_string()` was deprecated in ggplot2 3.0.0.
ℹ Please use tidy evaluation ideoms with `aes()`
```
But I think we can live with that at the moment until somebody knows how to fix that…